schemas/rivers_affected_area_py.yml

```diff
@@ -0,0 +1,13 @@
+required:
+    - name_en
```

Comment:
Do we need this for rivers? (There's no schema defined in the summary of internal artefacts)

Reply:
No schema defined. Removed from code